index.html

```diff
@@ -234,5 +234,38 @@ <h3>Skills</h3>
         </ul>
       </div>
     </div>
-  </section>  
+  </section>
+  <footer class="form-body">
+    <div class="card-contact">
+      <h2 class="contact-text">Contact me</h2>
+
+      <p class="contact-para">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/mpzbdpra" method="post">
+        <ul class="form-content">
+          <li>
+            <label for="name"></label>
+            <input class="form-list" id="name" type="text" name="user_name" placeholder="First Name" maxlength="30"
+              required />
```

Comment:
- [ ] I recommend you to adjust the placeholder so that it started at the same place where users will start writing, currently because the text holder starts too far, there is much difference:
<img width="267" alt="Screenshot 2022-02-15 at 19 05 24" src="https://user-images.githubusercontent.com/77038610/154089259-ce4ada59-1536-4911-ba47-a94fe99caffe.png">

Reply:
Thanks for the review. I have done the changes